build/build.sh

```diff
@@ -31,6 +31,7 @@ if [[ -z "$OUTPUT_DIR" ]] || [[ -z "$WMCO_SEMVER" ]] ; then
 fi
 
 PACKAGE="github.com/openshift/windows-machine-config-operator"
+MAIN_PACKAGE="${PACKAGE}/cmd/operator"
```

Comment:
MAIN_PACKAGE is a bit of a misnomer, this can be renamed to something like WMCO_CMD_DIR